### ISSUE TITLE
Avoid multiple (unnecessary) layout calls 

### ISSUE THREAD

**Issue by uruuru** [eclipse/elk#309]
Currently the layout button, and thus the `LayoutHandler#execute()` method, invokes a layout call every time it is pressed. We could add an option that avoids simultaneous layouts of the same diagram, for instance by scheduling the layout only if no new layout request has been issued for the same diagram.

**Comment by spoenemann** [comment]
We already have some code that checks for multiple operations on the same diagram, see `ElkServicePlugin.getRunningOperations()`. However, operations are canceled only _after_ a layout has been computed.

**Comment by uruuru** [comment]
Can't this yield lost updates when elements are added to a graph in the meantime? Also, numerous layout runs can start before any of them is canceled.

**Comment by spoenemann** [comment]
Yeah, it makes more sense to cancel old runs as soon as new ones arrive.
